internal/test/e2e/e2e_test.go

```diff
@@ -408,8 +407,47 @@ func sampleClusterTests(t *testing.T, kubectl *kubectlContext, sampleClusterPath
 				return err
 			}
 			// Don't assert on exact members, just that we have three of them.
-			if len(strings.Split(members, " ")) != 3 {
-				return errors.New(fmt.Sprintf("Expected etcd member list to have three members. Had %d.", len(members)))
+			numMembers := len(strings.Split(members, " "))
+			if numMembers != 3 {
+				return errors.New(fmt.Sprintf("Expected etcd member list to have three members. Had %d.", numMembers))
+			}
+			return nil
+		}, time.Minute*2, time.Second*10)
+		require.NoError(t, err)
+	})
+
+	t.Run("ScaleUp", func(t *testing.T) {
+		// Attempt to scale up to five nodes
```

Comment:
nit: `kubectl = kubectl.WithT(t)` here would ensure that the `kubectl.Scale` log messages are displayed with this sub-test rather than with the parent.